### PR TITLE
Modified Chat for Greeters

### PR DESCRIPTION
**Pull request by misslivirose**
Test plan: 
Attach https://hifi-content.s3.amazonaws.com/liv/Production/Start/loadSupportApp.js to an entity in a domain
Confirm that users around the entity are able to communicate via the SUPPORT app

### CODE REVIEW

**Comment by jazmincano** [comment]
This has been working perfectly as expected. Greeters have been using this with users and it's been great!

File: DomainContent/Start/html/ChatPage.html [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+              <input type="text" class="ChatInputText" id="ChatInputText" size="256" maxlength="256" />
+            </div>
+
+        </div>
+
+    </body>
+
+    <script>
+
+        //console.log("ChatPage: loading script...");
```
Comment (elisa-lj11) [inline DomainContent/Start/html/ChatPage.html]: remove print statements

File: DomainContent/Start/html/ChatPage.html [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+                //console.log("updateChatLog", i, a, displayName, message);
+                if (avatarID) {
+                    receiveChatMessage(avatarID, displayName, message, data);
+                } else {
+                    logMessage(message);
+                }
+            }
+        }
+
+        // Call this no every keystroke.
```
Comment (elisa-lj11) [inline DomainContent/Start/html/ChatPage.html]: *on

File: DomainContent/Start/html/ChatPage.html [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+                    .data('chat-message', message)
+                    .data('chat-data', data)
+                    .appendTo($ChatLog);
+            var $logLineDisplayName =
+                $('<span/>')
+                    .addClass('ChatLogLineDisplayName')
+                    .text(displayName + ': ')
+                    .on('mouseenter', function(event) {
+                        identifyAvatar(avatarID);
+                        //event.cancelBubble();
```
Comment (elisa-lj11) [inline DomainContent/Start/html/ChatPage.html]: remove commented out code

File: DomainContent/Start/html/ChatPage.html [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+            EventBridge.emitWebEvent(
+                JSON.stringify({
+                    type: "EmptyChatMessage",
+                    data: null
+                }));
+        }
+
+        // The user entered a non-empty chat message.
+        function handleChatMessage(message, data) {
+            //console.log("handleChatMessage", message);
```
Comment (elisa-lj11) [inline DomainContent/Start/html/ChatPage.html]: remove print out

File: DomainContent/Start/html/ChatPage.html [hunk 5]
```diff
@@ -0,0 +1,511 @@
...
+                    type: "FaceAvatar",
+                    avatarID: avatarID,
+                    displayName: displayName
+                }));
+        }
+
+        // Let's get this show on the road!
+        function main() {
+
+            //console.log("ChatPage: main");
```
Comment (elisa-lj11) [inline DomainContent/Start/html/ChatPage.html]: remove print

File: DomainContent/Start/html/ChatPage.html [hunk 6]
```diff
@@ -0,0 +1,511 @@
...
+
+            $ChatLog = $('#ChatLog');
+            $ChatInputText = $('#ChatInputText');
+
+            // Whenever the chat log resizes, or the input text gets or loses focus, 
+            // scroll the chat log to the last line.
+            $ChatLog.on('resize', function(event) {
+                //console.log("ChatLog resize", $ChatLog, event);
```
Comment (elisa-lj11) [inline DomainContent/Start/html/ChatPage.html]: remove print

File: DomainContent/Start/html/ChatPage.html [hunk 7]
```diff
@@ -0,0 +1,511 @@
...
+                scrollChatLog();
+            });
+            $ChatInputText.on('focus blur', function(event) {
+                //console.log("ChatInputText focus blur", $ChatInputText, event);
```
Comment (elisa-lj11) [inline DomainContent/Start/html/ChatPage.html]: remove print

File: DomainContent/Start/html/ChatPage.html [hunk 8]
```diff
@@ -0,0 +1,511 @@
...
+                }
+            });
+
+            // Start out with the input text in focus.
+            $ChatInputText.focus();
+
+            // Hook up a handler for events that come from hifi.
+            EventBridge.scriptEventReceived.connect(function (message) {
+
+                //console.log("ChatPage: main: scriptEventReceived", message);
```
Comment (elisa-lj11) [inline DomainContent/Start/html/ChatPage.html]: remove print

**Pull request merged**